scripts/renode_tests/BIOS.robot

```diff
@@ -0,0 +1,23 @@
+*** Settings ***
```

Comment:
I think these files should probably be under `./tests/renode/` rather than `./scripts/renode_tests`?

Reply:
moved